charts/calhoun/templates/configmaps/proxy-apache-httpd-configmap.yaml

```diff
@@ -0,0 +1,108 @@
+{{- if .Values.proxy.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.name }}-proxy-configmap
+  labels:
+    {{- include "calhoun.labels" . | nindent 4 }}
+data:
+  apache-httpd-proxy-config: |-
```

Comment:
I think the apache proxy is doing https resolution with the broadinstitute.org wildcard cert and calhoun is just listening on http. We could probably plug in the cert to calhoun as an alternative but I figured this was slightly easier.

Reply:
The proxy also integrates with `tcell` I think?

Reply:
Hi Rob, sorry I lost track of this in some of the chaos from Enkh leaving. If we just need the apache proxy for TLS termination, we do have some cleaner ways of handling that in the kubernetes world. 

The reason I'm hesitant to add it here unless necessary is that it is a bit of black box that isn't really well understood by most in the org currently. So I'm trying to avoid introducing new dependencies on it unless absolutely necessary (We hope to replace it at some point but I don't see that in the immediate future). 

The point about tcell is a good one though. If we need that for Calhoun then just using the proxy might be the best solution.

Reply:
I'm retracting my previous comment here, this was a bit of a  misunderstanding on my part. Sticking with the apache proxy is the best way forward here.